apps/token-e2e/src/integration/rewards.cy.js

```diff
@@ -0,0 +1,41 @@
+import navigation from '../locators/navigation.locators';
+import rewards from '../locators/rewards.locators';
+
+context('Rewards Page - verify elements on page', function () {
+  before('navigate to rewards page', function () {
+    cy.visit('/')
+      .get(navigation.section)
+      .within(() => {
+        cy.get(navigation.rewards).click();
+      });
+  });
+
+  describe('with wallets disconnected', function () {
+    it('should have REWARDS tab highlighted', function () {
+      cy.get(navigation.section).within(() => {
+        cy.get(navigation.rewards).should('have.attr', 'aria-current');
+      });
+    });
+
+    it('should have rewards header visible', function () {
+      cy.get(rewards.pageHeader)
+        .should('be.visible')
+        .and('have.text', 'Rewards');
+    });
+
+    it('should have epoch warning', function () {
+      cy.get(rewards.warning)
+        .should('be.visible')
+        .and(
+          'have.text',
+          'Rewards are credited 5 minutes after the epoch ends.This delay is set by a network parameter'
```

Comment:
comment (non-blocking): this text is generated based on the network parameter that is being set. As we are using capsule we have control of this but if the param changes the test breaks. I don't know that it needs a solution but perhaps something worth mentioning.

Reply:
Yes, I am aware of that. If we decide to modify this parameter we will just change it to env variable or sth.